assets/bower_components/iron-icon/iron-icon.html

```diff
@@ -8,6 +8,7 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 
+
```

Comment:
I messed with them to find out why they don't work. Unfortunately, I did not think in advance to try this in a different branch. There are no intentional updates to them.